storage/innobase/fil/fil0fil.cc

```diff
@@ -665,8 +665,15 @@ static bool fil_node_open_file(fil_node_t* node)
 					.physical())));
 	}
 
+#ifdef MYSQL_SERVER
```

Comment:
We do not recompile Innodb to build mariabackup. the "in mariabackup" condition can  be detected at runtime with
srv_operation != SRV_OPERATION_NORMAL